2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,10 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(arr) {
+  let A = arr.filter(x=>x[0]=='A')
```

Comment:
variable name should not start with a capital letter and could be more readable I.E. namesStartingWithA.
Could you use a suggestion made [here](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week4/pull/160/files#r1046458311)?

Reply:
changed it to namesStartingWithA